### DISCUSSION TITLE
Why comment #CONFIG_ETH_MCUX_RX_BUFFERS=6, #CONFIG_ETH_MCUX_TX_BUFFERS=8 in MIMXRT1060_EVK

### DISCUSSION THREAD

**Discussion by YMont** [zephyrproject-rtos/zephyr#66591] [Q&A] [answered]
**Describe the bug**
Hi All, I used RT1060_EVK gPTP SDK that I found `mimxrt1060_evk.conf` two feature are commented.
```
#CONFIG_ETH_MCUX_RX_BUFFERS=6
#CONFIG_ETH_MCUX_TX_BUFFERS=8
```
I'm curious why would do that ? Because it is different with RT1050 configuration.

Thanks a lot

**Comment by DerekSnell** [comment]
Hi @YMont ,
I converted your question to a Discussion, since you are not reporting an Issue.  However, I am not sure yet why those 2 lines are commented out.  I will investigate.  Thank you

**Comment by DerekSnell** [comment, marked as answer]
Hi @YMont ,
Discussing with our team who tested this gPTP sample, these commented Kconfig lines in `mimxrt1060_evk.conf` appear to be a mistake, but they are trivial.

`CONFIG_ETH_MCUX_RX_BUFFERS` [defaults to 6](https://github.com/zephyrproject-rtos/zephyr/blob/44af61b865d859e2e68f88d73ece1d5b63e2a383/drivers/ethernet/Kconfig.mcux#L59), same as the commented line.

And `CONFIG_ETH_MCUX_TX_BUFFERS` defaults to 1.  Our testing found more TX buffers were not required for this sample.

Best regards

**Comment by YMont** [comment]
Hi @DerekSnell,
Apologies for the delay. Should I enable it when I building gPTP sample?

Best regards
